content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,104 @@ aliases:
 keywords: [performance,scalability,scale,benchmarks]
 ---
 
-We follow a four-pronged approach to Istio performance characterization, tracking and improvements:
+Istio makes it easy to create a network of deployed services with rich routing,
+ load balancing, service-to-service authentication, monitoring, and more. It
+ does so without any changes to the application code. Istio strives to provide
+ these benefits with minimal resource overhead. Istio also aims to support very
+ large meshes with high request rates while adding minimal latency.
 
-* Code level micro-benchmarks
+## Data plane performance
 
-* Synthetic end-to-end benchmarks across various scenarios
+Istio consists of components that run on the data plane and components that
+ program the data plane, i.e. control plane. Data plane and control plane have
+ distinct performance concerns.
 
-* Realistic complex app end-to-end benchmarks across various settings
+### Latency
 
-* Automation to ensure performance doesn't regress
+Istio injects a sidecar proxy on the data path therefore latency is an important
+ consideration. Istio adds an auth and mixer filter to the proxy. Every
+ additional filter adds to the path length inside the proxy and affects latency.
 
-## Micro benchmarks
+Istio proxy collects raw telemetry data after the response is sent back to the
+ client. Time spent collecting raw telemetry for a request does not contribute
+ to the total time taken to complete that request. However, since the worker
+ thread is busy doing this work, it will not start work on the next request
+ immediately. This adds to the queue wait time of the next request affecting
+ average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-We use Go’s native tools to write targeted micro-benchmarks in performance sensitive areas. Our main goal with this approach is to provide easy-to-use micro-benchmarks that developers can use to perform quick before/after performance comparisons for their changes.
+Inside the mesh, a request traverses the client side proxy and then the server
+ side proxy. Two proxies on the data path add about `10ms` to the p99 latency at `1k rps`.
```

Comment:
client-side
server-side

Reply:
I'm a bit torn on whether or not we should be reporting the times for 2 proxies or 1 proxy. I understand the frankness of quoting numbers for 2 proxies...but I'm not sure people really think about that.  When they say "How much latency does the proxy add," I think they are thinking about 1 trip through the proxy.  WDYT?

Reply:
Question is how much latency does the mesh add usually. But PMs can decide :-)

Reply:
I added both numbers.